libmscore/drumset.cpp

```diff
@@ -162,6 +194,31 @@ int Drumset::prevPitch(int ii) const
       return 0;
       }
 
+//---------------------------------------------------------
+//   findVariant
+/// find a variant for the given pitch with matching chord articulation and tremolo
+//---------------------------------------------------------
+
+DrumInstrumentVariant Drumset::findVariant(int p, const QList<Articulation*> articulations, Tremolo* tremolo)
+      {
+      DrumInstrumentVariant div;
+      auto vs = variants(p);
+      for (auto v : vs) {
```

Comment:
Since it is a PR for 2.3, we decided to avoid c++11 `for` cycles and `auto` using, afair.

Reply:
Not really. It's not discouraged at all, we have a lot of them already but it's warmly advised in new code and in master, migrating to for is advised when touching code.